docs/sources/ingest-and-analyze-profile-data/analyze-profiles/_index.md

```diff
@@ -3,96 +3,35 @@ title: Analyze app performance using Pyroscope
 menuTitle: Analyze app performance
 description: How to use the Pyroscope UI to analyze performance of your applications.
 weight: 40
+draft: true
```

Comment:
This page needs more information about how to use Pyroscope to analyze content. Instead of holding up this PR, I've set this page to draft: true so the content won't be published.

Note that if you do a local build with make docs, this page won't show up. You have to remove this line for it to show up in preview builds. 

Doc issue for this task: https://github.com/grafana/pyroscope/issues/2856